plugins/pubIds/doi/pages/doiManagement/DOIManagementHandler.inc.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @file plugins/pubIds/doi/pages/doiManagement/DOIManagementHandler.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2003-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class DOIManagementHandler
+ * @ingroup pages_doiManagement
+ *
+ * @brief Handle requests for DOI management functions.
+ */
+
+import('classes.handler.Handler');
+import('plugins.pubIds.doi.classes.DoiListPanel');
+
+class DOIManagementHandler extends Handler {
+	const ENABLE_ISSUE_DOI = 'enableIssueDoi';
+	const ENABLE_PUBLICATION_DOI = 'enablePublicationDoi';
+	const ENABLE_REPRESENTATION_DOI = 'enableRepresentationDoi';
+
+	public $_isBackendPage = true;
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->addRoleAssignment(
+			[ROLE_ID_MANAGER, ROLE_ID_SITE_ADMIN],
+			['index', 'management']
+		);
+	}
+
+	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	public function authorize($request, &$args, $roleAssignments) {
+
+		import('lib.pkp.classes.security.authorization.PolicySet');
+		$rolePolicy = new PolicySet(COMBINING_PERMIT_OVERRIDES);
+
+		import('lib.pkp.classes.security.authorization.RoleBasedHandlerOperationPolicy');
+		foreach ($roleAssignments as $role => $operations) {
+			$rolePolicy->addPolicy(new RoleBasedHandlerOperationPolicy($request, $role, $operations));
+		}
+		// TODO: See if problem re: warning, "Expected parameter of type 'AuthorizationPolicy', 'PolicySet' provided
+		$this->addPolicy($rolePolicy);
+
+		return parent::authorize($request, $args, $roleAssignments);
+	}
+
+	/**
+	 * Displays DOI management index page
+	 *
+	 * @param array $args
+	 * @param PKPRequest $request
+	 */
+	public function index($args, $request) {
+		$this->management($args, $request);
+	}
```

Comment:
No need to have `index` and `management`. Let's just put it all under `index` so the URL is `/doiManagement`